src/components/ClusterNodes/NodeDetailsList.tsx

```diff
@@ -452,11 +467,20 @@ export default function NodeDetailsList({
         return '-'
     }
 
+    const clusterNodeClickEvent = (nodeData, column) => {
+        const url = `${match.url}/${nodeData[column.value]}`
+        return () => {
```

Comment:
Why are we returning this?

Reply:
naming could be better but this acts like a higher order function. The returned function is the actual handler. Earlier it was written as an arrow function in the onClick field